web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -951,17 +951,48 @@ span.upsell-price .tax {
   min-height: 0;
 }
 
+/**.front-page-luova-block:before {
+  content: '';
+  background: url("../assets/LuovaLogo.png");
+  background-repeat: no-repeat;
+  background-size: 150px;
+  position: absolute;
+  left: 300px;
+  top: 40px;
+  height: 100%;
+  width: 100%;
+}
+
+.front-page-luova-block:after {
+  content: '';
+  background: url("../assets/VCISLogo.png");
+  background-repeat: no-repeat;
+  background-size: 150px;
+  position: absolute;
+  left: 300px;
+  top: 160px;
+  height: 100%;
+  width: 100%;
+}*/
```

Comment:
Delete all this. No point in having commented out code in there. If you want to keep it for personal reference, put it on a blog or word doc somewhere.